generators/java/sdk/versions.yml

```diff
@@ -1,3 +1,24 @@
+- version: 2.39.6
```

Comment:
nit: since this is a feature, it should increment the minor version, not the patch version